### ISSUE TITLE
15 minutes SlotDuration and 1h TitleFormat

### ISSUE THREAD

**Issue by LMarcinkowski** [vkurko/calendar#372]
Hi,

Is it possible to achieve something like this:
![Zrzut ekranu 2024-11-5 o 07 56 19](https://github.com/user-attachments/assets/26616024-074c-4540-bd8c-57d06bcc11f1)

15 minutes SlotDuration but with one 1h header in `resourceTimelineDay` view?

**Comment by vkurko** [comment]
There is no such setting at the moment, but I think it would be very useful. Do you know if there is anything suitable in FullCalendar?

**Comment by LMarcinkowski** [comment]
Yes, it is in FullCalendar: 
https://fullcalendar.io/docs/timeline-custom-view-demo
https://codepen.io/pen?editors=0010

**Comment by inu-web** [comment]
Which amount of one-time sponsorship would it need to support this feature?
Maybe we could find a few people who'll sponsor it together?

**Comment by vkurko** [comment]
Thank you very much for your willingness to sponsor the development. Let's see what comes of this. I have added a one-time payment option for this feature request on the [sponsors](https://github.com/sponsors/vkurko?frequency=one-time) page. If anyone wants to donate part of the amount, they can change the amount to a smaller one in the input field.

**Comment by heresy60** [comment]
If I sponsor, will I know when it will be completed?

**Comment by vkurko** [comment]
Sorry, right now the priority is the transition to Svelte5 and I will be able to give some estimates for other sponsored feature requests only after this transition is completed.

**Comment by heresy60** [comment]
Now that the transition is complete, does sponsorship allow for feature development?

**Comment by vkurko** [comment]
> Now that the transition is complete, does sponsorship allow for feature development?

Yes, this is welcome now. I'll have some time to implement the feature next week.

**Comment by vkurko** [comment]
In v4.1.0 you can use [slotLabelInterval](https://github.com/vkurko/calendar?tab=readme-ov-file#slotlabelinterval) option in `resourceTimeline` views to get the desired result.

I've prepared a [demo](https://jsbin.com/nijegagidu/edit?html,output). The key settings are:

```js
slotDuration: '00:15',
slotLabelFormat: {hour: 'numeric'},
slotLabelInterval: '01:00',
slotWidth: 16,
```

and CSS:

```css
.ec-timeline .ec-time, .ec-timeline .ec-line {
    width: 16px;
}
.ec-timeline .ec-time {
    overflow: visible;
}
.ec-timeline .ec-time time {
    display: inline-block;
    width: 64px;
    text-align: center;
}
```
